src/components/2-Monitoring/monitoringText.vue

```diff
@@ -19,10 +19,9 @@
             paragraph11: "To visualize how gaps in temperature observations in space and time might impact our understanding of stream temperature dynamics in the DRB, we can represent the availability of data as a matrix with two axes: <span style=\"font-weight: 700; font-style: italic\">space</span> and <span style=\"font-weight: 700; font-style: italic\">time</span>.",
             title4: "Variability in Water Temperature",
             paragraph12: "Another complexity of predicting temperature dynamics in the basin is that not all streams behave the same way. Features of the landscape can have large effects on water temperature that result in different temperature patterns. Groundwater inputs, reservoirs, and urban areas all have distinct temperature signatures.",
-            paragraph13: "If we look at a single year of data, we can see that stream temperatures are not uniform throughout the basin.",
-            paragraph14: "In the summer of 2019, an urban reach in Philadelphia was the warmest stream reach in the basin, while a reach below the Beltzville Resevoir was one of the coolest. These reaches aren't that far from each other as the crow flies, but they experience very different seasonal temperature variations.",
-            paragraph15: "The matrix below shows a year of actual temperature data from all sites on medium to large streams in the DRB.",
-            paragraph16: "At this point, we know that stream temperatures on any given day vary throughout the basin and that the amount of data we have to work with varies in time and space. To predict what the temperature will be in a given stream, or how temperatures will change over time, we can use a model. But how can we create robust models that account for diverse temperature dynamics and make accurate predictions in places and times where we don't have a lot of information? Visit the Model chapter to learn about innovative modeling techniques to predict water temperature."
+            paragraph13: "If we look at a single year of data, we can see how different stream temperature can be across the basin. Each line in the figure below represents temperature observations from one river reach in the DRB in 2019. In July, mean stream temperature across the basin differed by as much as 26 degrees on a single day. Additionally, you can see drastically different warming and cooling patterns, particularly below reservoirs where stream temperature can deviate from the normal seasonal patterns.",
+            paragraph14: "Hover over a river reach on the map below to highlight temperature dynamics of a single reach. Alternatively, hover over a month on the line graph to view the average monthly temperature displayed on the map.",
```

Comment:
@hcorson-dosch -- I debated about what to say about reservoirs with this new figure since we're not specifically calling out one line to show reservoirs or other dynamics. What do you think about lines 22 and 23?

Reply:
I think that's good -- I like the part about res temps deviating from normal seasonal patterns. We may be able to use line 23 as the subtitle to the visual (within the black box), instead of as a paragraph -- I can get it all in there tomorrow and fine tune as needed